src/periscope.ts

```diff
@@ -295,10 +303,38 @@ export const periscope = () => {
       ...rgMenuActionsSelected,
       ...rootPaths,
       ...config.addSrcPaths,
+      ...(extraFlags || []),
       ...excludes,
     ];
 
-    return `rg '${value}' ${rgFlags.join(' ')}`;
+    return `"${rgPath}" '${value}' ${rgFlags.join(' ')}`;
+  }
+
+  // extract rg flags from the query, can match multiple regex's
+  function extraRgFlagsFromQuery(query: string): {
```

Comment:
Would it be useful if we use a regex to detect any appended flags on the query? 

Perhaps a regex such as: `const paramRegex = /\s(--?\w+.*)$/;`

This would remove the queryParam config requirement and provide a way to provide any rg flags.

Reply:
Yeah that's another approach to add parameters to `rg`. The two downsides I see are:
- Searching for the same pattern used in that regex (the edge case you mentioned)
- The user needs to type more characters to add the `rg` parameters from the query

I prefer having the list of parameters matches customisable to allow me to type as few chars as needed.
In my example above:
```json
{
      // filter the results to a folder
      // Query: redis -m module1 => rg 'redis' -g '**/*module1*/**'
      "param": "-g '**/*$1*/**' -g '!**/node_modules/**'",
      "regex": "^(.+) -m ([\\w-_]+)$"
    },
```

Just by adding `-m folder` to the query I'm adding two parameters to `rg`, which is is powerful.

Reply:
Ah ok got it, that makes sense me